TriblerGUI/widgets/trustpage/js/radial_view/radial_simulation.js

```diff
@@ -14,10 +14,10 @@ function RadialSimulation(options) {
     var defaults = {
         center_x: 0,
         center_y: 0,
-        alpha_decay: 0,
-        radial_force_strength: 0.2,
+        alpha_decay: 0.01,
```

Comment:
The animation very slowly dies out (when everything is already positioned correctly) so d3 stops calculating positions. This improves performance.